Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -160,13 +199,14 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
         for icon in self.iconImages! {
             icon.isHidden = !shouldShowInfo
         }
+        self.learnMoreButton?.isHidden = !shouldShowInfo
         self.scrollView?.isScrollEnabled = shouldShowInfo
         self.infoButton?.isHidden = shouldShowInfo
         self.navigationFooter?.shouldShowShadow = shouldShowInfo
     }
     
-    /// The function that is called when the info button is tapped.
-    override open func showLearnMore() {
+    /// Function called when the upper-right info icon action button is tapped
+    @objc open func showFullTaskInfo() {
```

Comment:
If you have mapped this to the reviewInstructions button in the nib, then you should override `showReviewInstructions()` instead of learn more.

Reply:
Note to self: The infoButton icon is no longer hooked up to an action b/c it's *very* specific to this particular implementation.